appveyor.yml

```diff
@@ -95,6 +128,7 @@ build_script:
 
 cache:
   - build-cmake
+  - build-cmake-vs2019
```

Comment:
Also by some reason cache is not used (cmake takes 12min over 3min, and this is due to checking of various flags/functions and so on) for the cmake builds after the [second build](https://ci.appveyor.com/project/libevent/libevent/builds/28226712) of this PR, can you take a look too?

Reply:
In my fork project, the build time is very long only in the first build. It is normal for all subsequent builds.
Also in the first build, modifications to the matrix may cause partial caching to be invalid.

Another consideration is that saving cache is disabled by default in the Pull Request build.
As [appveyor document](https://www.appveyor.com/docs/build-cache/#saveupdate-cache-before-build-finishes) says：
> Note: By default, saving cache is disabled in Pull Request builds. Use Save build cache in Pull Requests checkbox on the General tab of project settings if you need to save cache during PR builds. This setting is not exposed in YAML to prevent unauthorized cache modifications.

Therefore, I suggest try it after merging the code.

Reply:
>Also in the first build, modifications to the matrix may cause partial caching to be invalid.

That wasn't the first build

>By default, saving cache is disabled in Pull Request builds

Hm, interesting, I never knew this, but if you will take a look at another PR, say [#916](https://ci.appveyor.com/project/libevent/libevent/builds/28267865) you will see, that it uses cache fine

I think that the cache is not used, because you changed the env for the builds and hence cache key is different for this PR, in this case everything is ok.